ui-admin/src/HomePage.tsx

```diff
@@ -43,17 +45,18 @@ function HomePage() {
                     aria-expanded="false" aria-label="configure study"/>
                   <div className="dropdown-menu">
                     <ul className="list-unstyled">
-                      <li>
+                      <li key={`delete-${portal.shortcode}-${study.shortcode}`}>
```

Comment:
Should this be `delete-${selectedPortal.shortcode}-${selectedStudy.shortcode}`?

Reply:
Oh I think this could actually go away

Reply:
it should not be selected* because then the keys for all of items in the list would change based on the click, but we don't really need a key here because there is no need for a memory in the list here.